hs/src/Reach/Simulator/Core.hs

```diff
@@ -546,7 +546,11 @@ instance Interp DLExpr where
             Just v -> return v
             Nothing -> possible $ "DLE_Interact: late api call for " ++ partName'
         Nothing -> do
-          suspend $ PS_Suspend (Just at) (A_Interact slcxtframes partName' str dltype args)
+          case dltype of
+            T_Null -> do
+              return V_Null
```

Comment:
No log?

Reply:
if the function input is a variable/identifier in the program, it will show up in the local state details 
if it's a literal, its value is visible in the source code